gn3/computations/datasets.py

```diff
@@ -37,21 +35,22 @@ def retrieve_trait_sample_data(dataset,
 
     if dataset_query:
         formatted_query = dataset_query % sample_query_values[dataset_type]
-        results = fetch_from_db_sample_data(formatted_query, mock.Mock())
+
+        results = fetch_from_db_sample_data(formatted_query, database)
```

Comment:
**nitpick: suggestion** Rename `fetch_from_db_sample_data'` to `fetch_sample_data`